README.md

```diff
@@ -0,0 +1,361 @@
+- Advantages of the combined model
+    - More freedom on ways to provide liquidity, planned expansion for limit orders
+    - The combination of pool liquidity and limit order liquidity provide users with a more enriched trading environment
+
+For details, see the [Liquidity Module Light Paper](doc/LiquidityModuleLightPaper_EN.pdf).
+
+## Installation
+
+### Requirements
+
+| Requirement | Notes            |
+| ----------- | ---------------- |
+| Go version  | Go1.15 or higher |
+| Cosmos SDK  | v0.42.4 or higher        |
+
+### Get Liquidity Module source code 
+
+```bash 
+$ git clone https://github.com/tendermint/liquidity.git
+$ cd liquidity
+$ go mod tidy
+```
+
+### Build
+
+```bash 
+# You can find the liquidityd binary inside build directory
+$ make build 
+```
+
+### Install
+```bash 
+$ make install 
+```
+
+## Usage of CLI Commands
+
+### Transactions
+
+`$ liquidityd tx liquidity --help`     
+
+```bash
+Liquidity transaction subcommands
+
+Usage:
+  liquidityd tx liquidity [flags]
+  liquidityd tx liquidity [command]
+
+Available Commands:
+  create-pool Create liquidity pool and deposit coins
+  deposit     Deposit coins to a liquidity pool
+  swap        Swap offer coin with demand coin from the liquidity pool with the given order price
+  withdraw    Withdraw pool coin from the specified liquidity pool
+```
+
+### Queries
+
+`$ liquidityd query liquidity --help`
+
+```bash
+Querying commands for the liquidity module
+
+Usage:
+  liquidityd query liquidity [flags]
+  liquidityd query liquidity [command]
+
+Available Commands:
+  batch       Query details of a liquidity pool batch
+  deposit     Query the deposit messages on the liquidity pool batch
+  deposits    Query all deposit messages of the liquidity pool batch
+  params      Query the values set as liquidity parameters
+  pool        Query details of a liquidity pool
+  pools       Query for all liquidity pools
+  swap        Query for the swap message on the batch of the liquidity pool specified pool-id and msg-index
+  swaps       Query all swap messages in the liquidity pool batch
+  withdraw    Query the withdraw messages in the liquidity pool batch
+  withdraws   Query for all withdraw messages on the liquidity pool batch
+```
+
+#### A detailed document on client can be found here. [client.md](doc/client.md)
+
+## Development
+
+### Test
+```bash 
+$ make test-all
+```
+
+### 1. Setup local testnet using script
+
+```bash
+# This will bootstrap a single testnet locally.
+# Note that config, data, and keys are created inside 
+# ./data/localnet folder and RPC, GRPC, REST ports are all open.
+$ make localnet
+```
+
+### 1.1 Broadcast transactions using CLI commands
+
+Some sample scripts are available in [scripts](https://github.com/tendermint/liquidity/tree/develop/scripts) folder, which will help you to test out the liquidity module interface.
+
+### 2. Setup local testnet manually
```

Comment:
```suggestion
### 2. Manually set up a local testnet
```

Reply:
Is there a reason why someone would NOT want to use the `make test-all` script? if there is, we should provide that information, otherwise, why would anyone want to do all of this work manually?